source/Octopus.Cli/Commands/Releases/CreateReleaseCommand.cs

```diff
@@ -288,6 +304,35 @@ async Task<ReleasePlan> AutoSelectBestReleasePlanOrThrow(ProjectResource project
                 $"{releasePlans.Except(viablePlans).Select(p => p.FormatAsTable()).NewlineSeperate()}");
         }
 
+        /// <summary>
+        /// if no release notes were provided on the command line, but the project has a template, then use the template
+        /// </summary>
+        async Task ReleaseNotesFallBackToDeploymentSettings()
+        {
+            if (!string.IsNullOrWhiteSpace(ReleaseNotes))
+                return;
+
+            // Continue using deprecated property on DeploymentSettings that exposes project for older server backwards compatibility
+#pragma warning disable 618
+            var projectReleaseNotes = project.ReleaseNotesTemplate;
```

Comment:
Although based on the client model used by this solution the `ReleaseNotesTemplate` should still be on the `ProjectResource`, this property is being deprecated. To maintain compatibility here we can assume the property is in the old location first, then (vcs aside), we will look via the `DeploymentSettings` if this is supported to cater for a future scenario where it may be removed from the `ProjectResource`